Failed to compile typescript when starting frontend mock:api server

### What steps did you take:

Followed the `README.md` in `frontend` to run the UI using `npm start`, then tried to start up the `mock:api` using `npm run mock:api` as directed.

### What happened:

```
 ✘  ~/Code/pipelines/frontend   alexla/fix-travis-url ●  npm run mock:api

> pipelines-frontend@0.1.0 mock:api /Users/alexla/Code/pipelines/frontend
> ts-node-dev -O '{"module": "commonjs"}' --project mock-backend/tsconfig.json mock-backend/mock-api-server.ts 3001

Using ts-node version 7.0.1, typescript version 3.7.5
[ERROR] 09:43:19 ⨯ Unable to compile TypeScript:
src/lib/RunUtils.ts(31,31): error TS2307: Cannot find module 'src/apis/experiment'.
src/lib/RunUtils.ts(127,7): error TS7006: Parameter 'ref' implicitly has an 'any' type.
```

### What did you expect to happen:

Guessing it should start up a mock API on port 3001.

### Environment:

```
 ✘  ~/Code/pipelines/frontend   alexla/fix-travis-url ●  node --version
v12.18.0
 ~/Code/pipelines/frontend   alexla/fix-travis-url ●  npm --version
6.14.4
```

How did you deploy Kubeflow Pipelines (KFP)? Trying to set up developer environment to poke around the frontend 😄 

KFP version: https://github.com/kubeflow/pipelines/commit/2fa6e2b7f6855cecb61ae881226dba20ad80ba7a

KFP SDK version: KFP SDK not affected.

### Anything else you would like to add:

/kind bug
/area frontend

--- a/frontend/mock-backend/tsconfig.json
+++ b/frontend/mock-backend/tsconfig.json
@@ -1,7 +1,7 @@
 {
   "compilerOptions": {
     "allowSyntheticDefaultImports": true,
-    "baseUrl": ".",
+    "baseUrl": "..",
     "outDir": "build/dist",
     "module": "esnext",
     "target": "es5",

